src/05-derivatives/04-functional-derivatives.md

```diff
@@ -69,7 +69,7 @@ In addition, the following fields apply to the entire file in all cases:
 | StartTime         | OPTIONAL. Start time in seconds in relation to the start of acquisition of the first volume in the corresponding imaging file (negative values are allowed). |
 
 Note that there are several differences with these fields in
-[Physiological and other continuous recordings](../04-modality-specific-files/06-physiological-and-other-continuous-recordings.md).
+[Physiological and other continuous recordings](../04-modality-specific-files/06-physiological-and-other-continous-recordings.md).
```

Comment:
It should be "continuous" in all of these references. If it's misspelled throughout, I can open a PR for this !

Reply:
The file name is misspelled. A PR would be greatly appreciated.